ripple/src/main/webapp/businessdetails.js

```diff
@@ -52,4 +52,172 @@ function removeColorFromRating(ratingType, rating, maxRating, elementIdPrefix, a
 /* TODO: Function will add new review to firestore, then refresh the page to see changes */
 function newReview() {
   location.reload();
+  clearLocalStorage(["starRating", "priceRating"]);
+}
+
+/* Load all business details on the body onload */
+function bdOnload() {
+  // TEMPORARY LOCAL STORAGE UNTIL MANAGE BUSINESS INFO PAGE IS COMPLETE
```

Comment:
Why does this rely on the manage business info page? wouldn't it just be drawing from firestore?

Reply:
Nothing to fix, just wondering

Reply:
My bad, I meant the Main page and the Business Gallery pages files need to first store the 'walkingTime,' 'walkingDist,' in local storage. I also need to access other localStorage values that are set from the landing page, but I'm declaring them here so that I can isolate my workflow here for now.

Reply:
Gotcha